src/utils/errorUtils.ts

```diff
@@ -23,28 +27,29 @@ function generateFieldErrorFromErrorDetail(
   fieldName: string,
   errorDetail: ExceptionDetail
 ) {
+  if (typeof fieldName !== "string") {
+    throw new Error("fieldName can be string only");
+  }
```

Comment:
I wonder if we can use assertion function here:
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#assertion-functions

Reply:
Wasn't aware of this function, looks useful for such cases 👍
I think we can remove this block completely. Because TS already checks the type and will throw an error if we try to pass a non-string value.